### ISSUE TITLE
The user can login without approval (test it right after registration without closing the window)

### ISSUE THREAD

**Issue by kevinbkk** [pH7Software/pH7-Social-Dating-CMS#450]
User manual approval has been activated but the user can still login without admin approval.

**Note: This only happens if the user login directly after registration. If the user try to login at a later time, he can't login without admin approval.**

This is a major problem, please give this issue a priority.

**Comment by polynamaude** [comment]
@kevinbkk Don't put pressure on people. You already been told that we are all volunteer. You wrote the following when opening your issue **This is a major problem, please give this issue a priority.**

**Comment by kevinbkk** [comment]
> @kevinbkk Don't put pressure on people. You already been told that we are all volunteer. You wrote the following when opening your issue **This is a major problem, please give this issue a priority.**

What's wrong with you dude? You're destroying the image of this script. Are you an official support? I wish there's a block option here, you're really annoying.

**Comment by polynamaude** [comment]
@kevinbkk You have already been told that no change has been made to the source code that could relate to this issue. You can look at the commits if you feel like it.

You seem to have the opinion that you are entitled to some form of support and we have the obligation to respond to your requests. What I will continue to repeat is that we are all volunteers. On a second level, you don't give us any information that could allow to reproduce exactly what you are saying.

The answer **just install the script and do as I say** is missing much of what's needed. There's no log, no information about what system you are running over, etc.

Do some research on Google about what does **differential diagnosis** means and you'll get a vague idea about how to isolate, reproduce, identify problems.

Pertaining to official support, maybe you should stick to Microsoft solutions and take distance from FOSS (Free-OpenSource-Software). Or pay someone for a support plan.

And for sure, if I could reproduce your problem then I'd be into investigating and fixing it if needed.

If you don't like my answers the don't take a look inside the Linux kernel mailing list because you'll see that I'm quite patient and far from rude.

So on a end note, go to https://www.portableapps.com/ and download CamStudio so you can make a video of what you are reporting. As for now, we can't either reproduce it or try to because it's far from clear. And some subtle differences in the system can sometime cause issues. It also arise that bugs in other library are discovered when we use them together with other solutions on specific platforms.

Here's the link for the portable version of  [CamStudio](https://portableapps.com/apps/utilities/camstudio_portable)

This is a not a rule unique to pH7CMS but to nearly all software support, we investigate only what we can reproduce. And this is one reason why WIndows sends all the information about your computer hardware and software when it goes into a BSDO (Blue Screen Of Death).

**Comment by pH-7** [comment]
@polynamaude On this one, @kevinbkk is right. Thanks this post with more details, he specified that it only happens  with "Stay Signed" ticked. 
I didn't test it with "Stay Signed" ticked. 

However, this bug is there for a while and doesn't only occur on the latest version.

**Comment by pH-7** [comment]
@kevinbkk Thanks very much for this! I really appreciate that you cam with more details. 
Indeed, I didn't test it with "Stay Signed" ticked.

This bug is present for several versions (not only for the latest version). I will fix it and it will be released in pH7Builder, v15.

Thanks!

**Comment by polynamaude** [comment]
@pH-7 I didn't have that bug when I tried it out, even with the "stay signed in". As you just told me it was around for a while, I checked again. My Web Application Firewall was doing some **automatic fix** on it.

Do you have any selenium test made for pH7CMS ? Could be nice to prevent those type of regression that do happens one in a while.

**Comment by polynamaude** [comment]
@pH-7 I'm not sure for how long it has been in this bug as I did some digging to see when was the last time those pieces of code where changed and it's been quite some time. So that it was somehow related to the latest upgrades felt pretty strange. Well those things happens when you put too much in the same tickets.

**Comment by pH-7** [comment]
Hey @polynamaude! No, I don't have any tests with selenium. I haven't really used Selenium TBH. Did you?

I agree with you. Codeception is great too, but once again, I haven't implemented anything like that to pH7CMS.

**Comment by pH-7** [comment]
That bug has been around for quite a moment! 😄 
I did some refactoring with the "RememberMe" feature, so it's actually good!

https://github.com/pH7Software/pH7-Social-Dating-CMS/commit/cb8c220a54d5472f3234042318b506d5a064771c

**Comment by pH-7** [comment]
Yes @polynamaude, it's strange. Normally, it shouldn't be related to that latest version (14.9).

**Comment by polynamaude** [comment]
@pH-7 I don't know much about Codeception but I'll take a look.
I used Selenium for automatic UI testing for conformity with one of our institutional client guidelines. It's a bit time taking to get running in the beginning because you have to script some common part of every test like the login process but once that is done then you can easily try every possible options on your website and even do some tricky things like making fake cookies.

I'll take a look at what could be done and I get back.
